.travis/validate-package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Script to validate our packed package
+
+npm pack
+tar -xzf solid-community-server-*.tgz
+pushd package
+
+# Check if our server can start at a given port
+node bin/server.js -p 8888 &
+pid=$!
+i=0
+until curl -s localhost:8888; do
+  sleep 1
+
+  # Try for at most 10 seconds, assume failure otherwise
+  let i++
+  if [ $i -gt 10 ]; then
+    echo "Server start timeout"
+    echo "  Server may have failed to start, or is running at an unexpected port."
+    kill -9 $pid
+    exit 1;
```

Comment:
Maybe want to save your exit code instead and output it at the end, such that cleanup always happens.

Reply:
The problem is that the server may not have produced an exit code. It might either still be starting, or running at a different port.

Reply:
I mean, instead of `exit 1` here, to do `FAILURE=1` and then at the very end of this script `exit $FAILURE;`.

Reply:
Ah, I wasn't sure why you'd want that, but it's probably because we don't have the cleanup. I'll change it.